packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port.py

```diff
@@ -29,6 +33,29 @@
 }
 
 
+def _raise_if_symlink_not_valid(path: Path) -> None:
+    if not path.is_symlink():
+        return
+
+    symlink_target_path = Path(os.readlink(path))
+    if symlink_target_path.is_symlink():
+        message = (
+            f"'{path}' is pointing to '{symlink_target_path}' "
+            "which is itself a symlink. This is not supported!"
+        )
+        log.error(message)
+        raise SymlinkToSymlinkNotSupportedException(message)
+
+    if symlink_target_path.is_absolute():
+        message = (
+            f"Absolute symlinks are not supported: "
+            f"{path} points to {symlink_target_path} "
+            "Try with relative symlinks!"
+        )
+        log.error(message)
+        raise SymlinkWithAbsolutePathNotSupportedException(message)
```

Comment:
ok can we ensure the exception is used correctly with the 2 symlinks? it will simplify a bit this code, and make the exception usage more clear

Reply:
I meant pass the symlink as a parameter in the exception. it will make the code more compact.

Reply:
OK, mange to refactor it properly